baselines/powerplatform.md

```diff
@@ -1,12 +1,22 @@
+**`TLP:CLEAR`**
+
 # CISA M365 Security Configuration Baseline for Power Platform
 
-Microsoft Power Platform is a group of applications comprised of alow-code application development toolkit, business intelligence software, a custom chat bot creator, and app connectivity software.  This Secure Configuration Baseline (SCB) provides specific policies to help secure Power Platform security.
+Microsoft Power Platform is a cloud-based enterprise group of applications comprised of a low-code application development toolkit, business intelligence software, a custom chat bot creator, and app connectivity software.  This Secure Configuration Baseline (SCB) provides specific policies to help secure Power Platform security.
```

Comment:
This paragraph does not match the word version. The word version is:

Microsoft 365 (M365) Power Platform is a group of applications comprised of a low-code application development toolkit, business intelligence software, custom chat bot creator, and app connectivity software. This secure configuration baseline (SCB) provides specific policies to help secure Power Platform.

Reply:
updated to match!